napytau/gui/ui_mockup.py

```diff
@@ -1,60 +1,142 @@
 import tkinter as tk
+from tkinter import Canvas
+
+import customtkinter
+from matplotlib.figure import Figure
+from matplotlib.backends.backend_tkagg import FigureCanvasTkAgg
 
 from napytau.cli.cli_arguments import CLIArguments
 from napytau.core.logic_mockup import logic
 
+customtkinter.set_appearance_mode("System")  # Modes: "System" (standard), "Dark", "Light"
+customtkinter.set_default_color_theme("blue")  # Themes: "blue" (standard), "green", "dark-blue"
+
+
+def plot(value, window) -> Canvas:
+    # the figure that will contain the plot
+    fig = Figure(
+                figsize=(3, 2),
+                dpi=100,
+                facecolor="white",
+                edgecolor="black"
+                 )
+    
+    # list of squares
+    y = [(i - 50) ** value for i in range(101)]
+    
+    # adding the subplot
+    plot1 = fig.add_subplot(111)
+    
+    # plotting the graph
+    plot1.plot(y)
+    
+    # creating the Tkinter canvas
+    # containing the Matplotlib figure
+    canvas = FigureCanvasTkAgg(fig, master=window)
+    canvas.draw()
+    
+    return canvas.get_tk_widget()
+
+class App(customtkinter.CTk):
```

Comment:
Warum genau erstellst du hier eine neue Klasse für das CustomTKinter Window? Ist dies notwendig? Werden wir das fortan immerzu machen müssen oder ist das nur so aus dem Grund das du hier zwei Fenster nacheinander öffnen willst

Reply:
Please try to stick to english.
Instantiating a class here is part of Tkinters widget system, you can see it as something akin to Class Components in React. This isn't strictly required by Tkinter, however i would suggest we use it when we start really implementing the ui, as this will make the code modular and easier to maintain.